e2e-tests/assistant-view.spec.ts

```diff
@@ -0,0 +1,116 @@
+import { test, expect } from '@playwright/test';
+
+test('Test fetching Kristiansund - Molde and loading more after first result', async ({
```

Comment:
What is the naming convention for these tests?

Reply:
Should start with should if we're following BDDish naming. I didn't start with that until next test tho and didn't update this one 😅